### PR TITLE
Add a note about container runtime

### PR DESCRIPTION
**Pull request by dmitryax**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,6 +95,21 @@ While this helm chart should work for other Kubernetes distributions, it may
 require additional configurations applied to
 [values.yaml](helm-charts/splunk-otel-collector/values.yaml).
 
+---
+**IMPORTANT**
+
+By default, the chart is configured to collect logs from k8s clusters with
+**docker** runtime which will be deprecated starting from Kubernetes
```
Comment (pmcollins) [inline README.md:102]:
```suggestion
a **docker** runtime, which will be deprecated starting with Kubernetes
```

File: README.md [hunk 2]
```diff
@@ -95,6 +95,21 @@ While this helm chart should work for other Kubernetes distributions, it may
...
+version 1.20.
+
+If your cluster is running with **contained** or **cri-o** runtime make sure to
```
Comment (pmcollins) [inline README.md:105]:
```suggestion
If your cluster is running with a **containerd** or **cri-o** runtime, make sure you
```

**Review by pmcollins — APPROVED** [review]
Just a couple of small grammar possibilities. I don't feel strongly about them, so take them or leave them 🙂. Although I think you meant `containerd` for the runtime?

**Review by flands — APPROVED** [review]
LGTM with Pablo's feedback incorporated

**Pull request merged**